website_signup_legal_page_required/__init__.py

```diff
@@ -0,0 +1,4 @@
+# -*- coding: utf-8 -*-
```

Comment:
copyright of all __init__file should be as in https://github.com/OCA/maintainer-tools/blob/master/template/module/__init__.py